service/worker/scanner/history/scavenger_test.go

```diff
@@ -67,7 +69,10 @@ func (s *ScavengerTestSuite) createTestScavenger(rps int) (*mocks.HistoryV2Manag
 	db := &mocks.HistoryV2Manager{}
 	controller := gomock.NewController(s.T())
 	workflowClient := history.NewMockClient(controller)
-	scvgr := NewScavenger(db, 100, workflowClient, ScavengerHeartbeatDetails{}, s.metric, s.logger)
+
+	dynamicConfigColleciton := dynamicconfig.NewNopCollection()
+	maxWorkflowRetentionInDays := dynamicConfigColleciton.GetIntProperty(0, domain.DefaultMaxWorkflowRetentionInDays)
```

Comment:
nit: use `dc.GetIntPropertyFn(common.DefaultMaxWorkflowRetentionInDays)` in test so no collection is needed.

Reply:
Sure, thanks!